_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
+
+## Usage
+
+See the following `.yaml` configuration file:
+
+```
+processor:
+    - otel_metrics_raw_processor
+```
+
+## Configuration
+
+The following table describes the different parameters used to configure histogram buckets and their default values.
+
+| Parameter | Default value | Description |
+| :---    | :---    | :---    |
+| `calculate_histogram_buckets` | `False` if not provided | |
+| `calculate_histogram_buckets` | `False` if not provided | |
+| `calculate_exponential_histogram_buckets` |  | |
+| `exponential_histogram_max_allowed_scale` | `10` if not provided | |
+
+
+If `calculate_histogram_buckets` is not set to `false`, the following JSON file will be added to every histogram JSON. If `flatten_attributes` is set to `false`, the JSON string format of the metrics does not change the attributes field. If `flatten_attributes` is set to `true`, the values in the attributes field are placed in the parent JSON object. The default value is `true`. See the following JSON example:
+
+```json
+ "buckets": [
+    {
+      "min": 0.0,
+      "max": 5.0,
+      "count": 2
+    },
+    {
+      "min": 5.0,
+      "max": 10.0,
+      "count": 5
+    }
+  ]
+```
+
+You can create detailed representations of histogram buckets and their boundaries. You can control this feature by using the following parameters in a YAML file:
+
+```yaml
+  processor:
+    - otel_metrics_raw_processor:
+        calculate_histogram_buckets: true
+        calculate_exponential_histogram_buckets: true
+        exponential_histogram_max_allowed_scale: 10
+        flatten_attributes: false
+```
+
+Each array element describes one bucket. Each bucket contains the lower boundary, upper boundary, and its value count. This is a specific form of denser OpenTelemetry representation that is a part of the JSON output created by the following plugin:
+
+<!--- Is this correct? Is this a plugin? It looks like a JSON file.--->
+
+```json
+ "explicitBounds": [
+    5.0,
+    10.0
+  ],
+   "bucketCountsList": [
+    2,
+    5
+  ]
+```
+
+
+If `calculate_exponential_histogram_buckets` is not set to `false`, the following JSON values are added to each JSON historgram:
```

Comment:
Why are these values added?

Reply:
Because `calculate_exponential_histogram_buckets` is by default true, those values will be added.